protocol/Cargo.toml

```diff
@@ -10,15 +10,18 @@ rust-version = "1.63.0"
 
 [features]
 default = ["std"]
+# High-level wrappers using futures traits.
 async = ["std", "futures/std"]
+# High-level wrappers using tokio traits - may affect MSRV requirements.
 tokio = ["std", "tokio/io-util"]
 std = ["alloc", "bitcoin/std", "bitcoin_hashes/std", "chacha20-poly1305/std", "rand/std", "rand/std_rng"]
 alloc = ["chacha20-poly1305/alloc"]
 
 [dependencies]
 futures = { version = "0.3.30", default-features = false, optional = true }
-# Must be under 1.39.0 due to MSRV 1.63.0 requirement.
-tokio = { version = ">=1.37.0, <1.39.0", default-features = false, optional = true }
+# The tokio feature may increase the MSRV beyond 1.63.0
+# depending on which version of tokio is selected by the caller.
+tokio = { version = "1", default-features = false, optional = true }
```

Comment:
FTR I've seen people saying that writing 1.0.0 is better since it conveys the intention. But I don't really care much.

Reply:
So same semantics, but 1.0.0 conveys something like "Starting exactly at this point version"? Just making sure I understand what I am signing up for here.

Reply:
Same semantics in the eyes of `cargo` but 1.0.0 conveys to some people "I didn't write '1' just because I was lazy, I really intended 1.0.0". (I personally don't agree with those people that they should care, but they do exist and perhaps making them happy is worth 4 extra characters. :))